pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython==0.29.21", "oldest-supported-numpy"]
+requires = ["setuptools", "wheel", "cython==0.29.24", "oldest-supported-numpy"]
```

Comment:
Maybe `cython~=0.29.24` is better? That way any patch release of 0.29 can be used (so 0.29.24 or later, but not 0.30 or later), which allows for cpython bugfixes.

```suggestion
requires = ["setuptools", "wheel", "cython~=0.29.24", "oldest-supported-numpy"]
```

Reply:
Very good point, as this also caused this issue in the first place.